Solutions/FalconFriday/Analytic Rules/DotNetToJScript.yaml

```diff
@@ -0,0 +1,42 @@
+id: 9f921513-65f3-48a2-ae7d-326c5901c55e
+name: Detect .NET to JS
+description: |
+  This query detects .NET being loaded from wscript or cscript to run .NET code, such as cactustorch and sharpshooter.
+  All based on the DotNetToJScript by James Foreshaw documented here https://github.com/tyranid/DotNetToJScript. 
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceImageLoadEvents
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Execution
+relevantTechniques:
+  - T1204
+query: |
+  DeviceImageLoadEvents 
+  | where FileName in~ ("mscoree.dll", "mscorlib.dll", "mscorlib.ni.dll") 
+  | where tolower(InitiatingProcessFileName) in ("wscript.exe", "cscript.exe", "mshta.exe")
```

Comment:
Format output project or summarize

Reply:
See previous points on `project` and `summarize`